We are using PAM via helper, do not issue following warnings:
xscreensaver: couldn't get password of "my_uid"
xscreensaver: couldn't get password of "root"
================================================================================
Index: driver/passwd-pwent.c
===================================================================
--- a/driver/passwd-pwent.c
+++ b/driver/passwd-pwent.c
@@ -187,7 +187,7 @@ get_encrypted_passwd (const char *user)
 
   if (!result &&
       (verbose_p
-# ifdef HAVE_PAM
+# if 1
        || 0
 # else
        || 1
